code/modules/projectiles/guns/ego_gun/he.dm

```diff
@@ -3,10 +3,12 @@
 	desc = "The small accessory remains like the wishes of a child who yearned for happiness."
 	icon_state = "prank"
 	inhand_icon_state = "prank"
-	special = "This weapon does 20 black melee damage."
+	special = "This weapon fires as fast as you pull the trigger. \
+			This weapon has an extremely slow melee. \
+			This weapon has knockback."
 	ammo_type = /obj/item/ammo_casing/caseless/ego_prank
 	weapon_weight = WEAPON_HEAVY
-	force = 20
+	force = 60
```

Comment:
You should not be using guns for melee at all.

Reply:
But the DPS is shit....